cmds/import_image/import_image_cmd.go

```diff
@@ -84,5 +114,28 @@ func (cmd *ImportImageCmd) Run() error {
 	cmd.Id = vgbdtgObject.Id
 	cmd.Uuid = vgbdtgObject.GlobalIdentifier
 
+	if cmd.Public {
+		execStmtRetryable := boshretry.NewRetryable(
+			func() (bool, error) {
+				id, err := vgbdtgService.CreatePublicArchiveTransaction(cmd.Id, cmd.PublicName, cmd.PublicNote, cmd.PublicNote, locations)
+				if err != nil {
+					return true, errors.New(fmt.Sprintf("There would be an active transaction in progress."))
+				}
+
+				cmd.Id = id
+				cmd.Uuid = ""
+
+				return false, nil
+			})
+		timeService := clock.NewClock()
+		timeoutRetryStrategy := boshretry.NewTimeoutRetryStrategy(5*time.Minute, 30*time.Second, execStmtRetryable, timeService, boshlog.NewLogger(boshlog.LevelInfo))
```

Comment:
Seems array can't be declared as a constant, got the following error if I declare it a constant:
```
cmds/import_image/import_image_cmd.go:34: const initializer []data_types.SoftLayer_Location literal is not a constant
```